Dockerfile

```diff
@@ -18,4 +15,4 @@ COPY ps-entry.sh /entrypoint.sh
 ENTRYPOINT ["/entrypoint.sh"]
 
 EXPOSE 3306
-CMD ["mysqld"]
+CMD ["mysqld","--user=mysql"]
```

Comment:
I've user=mysql in my.cnf
https://github.com/percona/percona-docker/blob/master/percona-server/ps-entry.sh#L112